From: Cole Robinson <crobinso@redhat.com>
Date: Tue, 13 Dec 2016 12:58:14 -0500
Subject: osdict: Don't return virtio1.0-net as a valid device name

We can't depend on libosinfo device names being valid libvirt network
model names, so use a whitelist

https://bugzilla.redhat.com/show_bug.cgi?id=1399083

Closes: #846356
---
 virtinst/osdict.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/virtinst/osdict.py b/virtinst/osdict.py
index 37426fd..38d626d 100644
--- a/virtinst/osdict.py
+++ b/virtinst/osdict.py
@@ -453,7 +453,7 @@ class _OsVariant(object):
         devs = self._os.get_all_devices(fltr)
         for idx in range(devs.get_length()):
             devname = devs.get_nth(idx).get_name()
-            if devname != "virtio-net":
+            if devname in ["pcnet", "ne2k_pci", "rtl8139", "e1000"]:
                 return devname
         return None
 
